Automattic/wp-calypso
# Command Palette: Use site hostname instead of window hostname

Fixes:
- https://github.com/Automattic/dotcom-forge/issues/5808

## Proposed Changes

* We use the `siteHostname` that we receive in the `commandPaletteConfig` instead of resolving it from the window object.

## Testing Instructions



* Apply the latest version of https://github.com/Automattic/jetpack/pull/35635 to your Atomic site
* Run install-plugin.sh command-palette add/command-palette-use-real-hostname in your sandbox
* Sandbox widgets.wp.com
* Open the command palette on site with a custom domain.
* You should see the `Manage DNS records` command and all other `isCustomDomain` commands.

## Pre-merge Checklist
